fe/fe-core/src/main/java/com/starrocks/qe/scheduler/FeExecuteCoordinator.java

```diff
@@ -62,11 +66,22 @@ public class FeExecuteCoordinator extends Coordinator {
     private final ConnectContext connectContext;
 
     private final ExecPlan execPlan;
+    private final QueryRuntimeProfile queryProfile;
 
 
-    public FeExecuteCoordinator(ConnectContext context, ExecPlan execPlan) {
+    public FeExecuteCoordinator(ConnectContext context,
+                                ExecPlan execPlan,
+                                List<PlanFragment> fragments,
+                                List<ScanNode> scanNodes,
+                                TDescriptorTable descTable) {
         this.connectContext = context;
         this.execPlan = execPlan;
+        if (descTable == null) {
+            throw new IllegalArgumentException("TDescriptorTable cannot be null");
+        }
+        JobSpec jobSpec = JobSpec.Factory.fromQuerySpec(context, fragments,
+                scanNodes, descTable, TQueryType.SELECT);
+        this.queryProfile = new QueryRuntimeProfile(connectContext, jobSpec, false);
```

Comment:
no need to new a empty queryProfile, StmtExecutor::buildTopLevelProfile which will keep fe planner profile, you should figure out why that profile  is lost

Reply:
The problem is about this:
![image](https://github.com/user-attachments/assets/0e795e09-2bcc-488e-bd16-fa78ad2863ab)


In the previous implementation, the task will not be executed. So the profile will be not be recorded.